pkg/store/bucket.go

```diff
@@ -1827,6 +1849,9 @@ func (it *bigEndianPostings) length() int {
 }
 
 func (r *bucketIndexReader) PreloadSeries(ids []uint64) error {
+	start := time.Now()
+	defer r.block.metrics.indexOperationDuration.WithLabelValues(opLoadSeries).Observe(float64(time.Since(start)))
```

Comment:
Should include the time it takes to fetch from cache? Currently it does. If yes, I would mention it in the metric description.